import successResponse from "../helpers/success.response";
import { customError } from "../helpers/customError";
const Auth = {
  signup: async () => {},
  signin: async () => {},
};
